server/ai_process.go

```diff
@@ -706,6 +707,84 @@ func submitSegmentAnything2(ctx context.Context, params aiRequestParams, sess *A
 	return resp.JSON200, nil
 }
 
+// TODO: CalculateTextToSpeechLatencyScore should be updated to reflect length of input tokens/output file size
```

Comment:
Are we going to do this as part of this PR or as a followup?

Reply:
The equivalent to output pixels for the TextToSpeech pipeline is input characters ( pricing per character ), so I think we can remove this comment as it is outdated.

In lieu of a comment to that effect, I think I will change the parameter name: `outPixels` -> `inCharacters`.